packages/opam-check-npm-deps/opam-check-npm-deps.1.0.0/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis:
+  "An opam plugin to check for npm depexts inside the node_modules folder"
+description:
+  "Provides the `opam check-npm-deps` command, which given an opam switch, gathers all the depexts belonging to the npm platform and their version constraints, and checks the `node_modules` folder to see if the constraints are satisfied."
+maintainer: ["Javier Chávarri"]
+authors: ["Javier Chávarri"]
+license: "MIT"
+homepage: "https://github.com/jchavarri/opam-check-npm-deps"
+bug-reports: "https://github.com/jchavarri/opam-check-npm-deps/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "reason" {>= "3.8.1" & < "4.0.0"}
+  "dune" {>= "3.8"}
+  "opam-client" {>= "2.1.2" & < "2.2.0"}
```

Comment:
It’s better to have at least 2.1.3 to have this patch: https://github.com/ocaml/opam/issues/4554
Also `< "2.2.0"` also includes 2.2.0~alpha which is probably not what you want. `< "2.2"` will remove that from the selected set
```suggestion
  "opam-client" {>= "2.1.3" & < "2.2"}
```